tests/components/pi_hole/test_init.py

```diff
@@ -69,6 +87,9 @@ async def test_setup_minimal_config(hass):
     assert hass.states.get("sensor.pi_hole_domains_blocked").state == "0"
     assert hass.states.get("sensor.pi_hole_seen_clients").state == "0"
 
+    assert hass.states.get("binary_sensor.pi_hole").name == "Pi-Hole"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5872960)